include/cantera/thermo/ThermoPhase.h

```diff
@@ -109,6 +109,11 @@ class ThermoPhase : public Phase
         return "ThermoPhase";
     }
 
+    //! String indicating the phase (liquid, vapor, etc.) of the phase
+    virtual std::string getPhase() const {
+        throw NotImplementedError("ThermoPhase::getPhase");
```

Comment:
Should other phase types overload this information as well?

Reply:
Yes, presumably any other multiphase capable phases should overload this method. I'm not sure how this should be handled for things like multi-species/multi-phase cases, where one species might be two-phase but another would be superheated vapor. So I only provided the implementation for PureFluid.

Reply:
True, some cases may be tricky. For others, this may be useful to auto-document assumptions for any parameterizations, even single-phase.

Reply:
I'm not sure what you mean by assumptions?

Reply:
Sorry, I guess that was cryptic. I meant assumptions for what phase is described; for some models it’s not immediately clear from the class name, even if the docstring ultimately clarifies things. I guess it boils down to #652: if this function is introduced for troubleshooting purposes, it could also help improve the overall documentation status. It shouldn’t be difficult to define a limited set of phase descriptors that can be used consistently throughout cantera.